docs/source/using/accessing-the-openshift-docker-registry.adoc

```diff
@@ -0,0 +1,44 @@
+= Accessing the OpenShift Docker Registry
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+OpenShift provides an integrated docker registry which can be used as a local registry.
+
+This helps to speed up the local development as images present in registry can be used for applications running in OpenShift.
+
+== Login to the docker registry 
+
+. Start minishift with the link:../command-ref/minishift_start{outfilesuffix}[`minishift start`] command.
+
+. Add the oc binary to the PATH environment variable.
+
+ $ export PATH=$PATH:~/.minishift/cache/oc/{openshift-version}
+
+. Export the environment variables that are required link:../using/reusing-docker-daemon{outfilesuffix}[to reuse the Docker daemon]
+
+ $ eval $(minishift docker-env)
+
+. Login to the docker registry
+
+ $ docker login -u developer -p `oc whoami -t` 172.30.1.1:5000 
+
+== Staging your application with the OpenShift registry
+
+This scenario is intended to illustrate how to test an application Docker image to a local instance of OpenShift platform that runs in Minishift.
+
+. Build the Docker image as usual.
+
+. Tag the image against the OpenShift registry with the hostname.
```

Comment:
"with the hostname" - that's not correct. We are not using a hostname here. We are using the _fixed_ internal OpenShift registry IP of 172.30.1.1.

It should in fact be mentioned somewhere, that usually the internal IP of the OpenShift registry is dynamic, but that _cluster up_ uses a fixed IP to allow for this use case.